src/Config.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Wearesho\Phonet;
+
+use GuzzleHttp\ClientInterface;
+
+/**
+ * Class Config
+ * @package Wearesho\Phonet
+ */
+class Config implements ConfigInterface
+{
+    use ConfigTrait;
+    use AuthorizationProviderTrait;
+    use ClientTrait;
+
+    public function __construct(
+        ClientInterface $client,
```

Comment:
Should be used as GuzzleHttp\ClientInterface, because `ClientInterface` is too common.